templates/init/controller/router.got

```diff
@@ -1,13 +1,15 @@
 package controller
 
 import (
-	"github.com/labstack/echo"
-	"github.com/labstack/echo/middleware"
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
 )
 
 //MapRoutes for the endpoints which the API listens for
 func MapRoutes(e *echo.Echo) {
 	g := e.Group("/{{.AppName}}/v1")
+	p := prometheus.NewPrometheus("echo", nil)
```

Comment:
Prometheus usage could be optional, using an environment variable like "UsePrometheus: USE_PROMETHEUS", boolean (default false)